packages/edit-site/src/components/header/fullscreen-mode-close/style.scss

```diff
@@ -8,22 +8,23 @@
 		align-items: center;
 		align-self: stretch;
 		border: none;
-		background: #23282e; // WP-admin gray.
```

Comment:
per the design, should we keep this as the background even when we're using the site icon?

Reply:
That's a great point 👍. I'll fix it and maintain the WP-admin gray even with the site icon.